examples/upload/src/controller.erl

```diff
@@ -0,0 +1,186 @@
+-module(controller).
+
+-export([
+    execute/2,
+	process/4,
+	resource_exists/1,
+	content_types_provided/1,
+	content_types_accepted/1,
+	allowed_methods/1,
+	post_is_create/1,
+	create_path/1
+]).
+
+execute(Req, Env) ->
+	{ok, Req, Env#{ cowmachine_controller => ?MODULE }}.
+
+process(<<"GET">>, _ContentType, _Accepted, Context) ->
+	{true, Context};
+process(<<"POST">>, _ContentType, _Accepted, Context) ->

```

Comment:
Maybe we have something the wrong way around in another example, or piece of code.

The arguments of process/4 are

``process(_Method, AcceptedCT, ProvidedCT, Context) ``

Where AcceptedCT is the content-type of the incoming payload, and ProvidedCT is the content-type of the response.

Reply:
I alredy change (simplify) this code example. I did not expect that is parameter so useful.